doc/03-omnifest/02-external.md

```diff
@@ -4,20 +4,16 @@
 
 These directives are only allowed within a [`otk.target.osbuild.<name>`](./01-directive.md#otktargetconsumername).
```

Comment:
If the directives are only allowed under a specific target, we could allow users to write this shother: `otk.external.osbuild.file-from-text`` -> `otk.external.file-from-text`` as we know that the context is `osbuild` from the  target. This would also ensure that users cannot use external commands that are not matching their target

Reply:
We could though I like the explicitness; I guess I'm about even on this so let's see what others feel like.

It'd *feel* weird to me if we had `otk.external.file-from-text` that does different things under different targets.

Reply:
I have no super strong opinion here but we should make sure that we do not allow using tools for targets that do not match the target target. 

I am not sure if e.g. different `otk.external.user` implementations depending on the target would be bad, I mean, they are clearly scoped under a
```
otk.target.osbuild:
 stages:
   - otk.external.users:
````
and in kiwi the same. But I don't really mind much, it just seems a bit redundant (both in terms of extra checks needed to avoid mixing targets and in extra typing for the users).